docs/devices.md

```diff
@@ -55,79 +17,108 @@ For mice, the DPI column specifies if the mouse's sensitivity is fixed (`-`),
-| M585             | 4.5   | yes     | -     |                                 |
-| M590             | 4.5   | yes     | -     |                                 |
-| M600 Touch       | 2.0   | yes     |       |                                 |
-| M705 Marathon    | 1.0   | yes     | -     | smooth scrolling                |
-| T400 Zone Touch  | 2.0   | yes     |       | smooth scrolling                |
-| T620 Touch       | 2.0   | yes     |       |                                 |
-| Performance MX   | 1.0   | yes     | R/W   | smooth scrolling                |
-| Anywhere MX      | 1.0   | yes     | R/W   | smooth scrolling                |
-| Anywhere MX 2    | 4.5   | yes     | R/W   | smooth scrolling                |
-| MX Master        | 4.5   | yes     | R/W   | smart shift                     |
-| Cube             | 2.0   | yes     |       |                                 |
-
-
-Mice (Nano):
-
-| Device           | HID++ | Battery | DPI   | Other supported features        |
-|------------------|-------|---------|-------|---------------------------------|
-| G700s            | 1.0   | yes     | -     | smooth scrolling                |
-| G700             | 1.0   | yes     | -     | smooth scrolling                |
-| V450 Nano        | 1.0   | yes     | -     | smooth scrolling                |
-| V550 Nano        | 1.0   | yes     | -     | smooth scrolling                |
-| VX Nano          | 1.0   | yes     | -     | smooth scrolling                |
-| M175             |       | yes     |       |                                 |
-| M185 (old)       | 4.5   | yes     | R/W   | smooth scrolling (note)         |
-| M185 (new)       | 4.5   | no      | R/W   | smooth scrolling (note)         |
-| M187             | 2.0   | yes     |       |                                 |
-| M215             | 1.0   | yes     |       |                                 |
-| M235             | 4.5   | yes     |       |                                 |
-| M305             | 1.0   | yes     |       |                                 |
-| M310             | 1.0   | yes     |       |                                 |
-| M315             |       | yes     |       |                                 |
-| M330             | ?.?   | yes     | ?     | smooth scrolling                |
-| MX 1100          | 1.0   | yes     | -     | smooth scrolling, side scrolling|
+You are able to read this feature using command-line interface of Solaar, but it is not possible to assign different keys.
+
+
+### Receivers:
+
+| USB ID    | Kind       | Max Paired Devices |
+------------|------------|--------------------|
+| 046d:c52b | Unifying   | 6                  |
+| 046d:c52f | Nano       | 2(?)               |
+| 046d:c532 | Unifying   | 6                  |
+| 064d:c534 | Nano(?)    | 2                  |
+| 064d:c539 | Lightspeed | 6                  |
+| 064d:c53a | Lightspeed | 6                  |
+| 064d:c53f | Lightspeed | 6                  |
```

Comment:
Lightspeed and Nano recivers have a max of 1 paired device. `c534` seems to be a nano receiver.

You can find more IDs here:
https://github.com/pwr-Solaar/Solaar/blob/master/lib/logitech_receiver/base_usb.py#L35

Reply:
I have a bunch of c534s and they all can have two paired devices.  They came in MK270 boxes containing an M185 mouse and a K270 keyboard.
It may be that some other nano receivers can only pair one device.
The c534 receivers have the feature (!) that they can only do a limited number of pairings.

Reply:
Hum, I am mistaken then. Keep the 2 there. Lightspeed receivers though definitely can only pair 1 device.

Reply:
I'll put in more receivers and adjust the values.